src/instrument/mongoose_instrument.erl

```diff
@@ -256,6 +318,10 @@ get_handlers(EventName, Labels) ->
 handle_event(Event, Labels, Measurements, {EventHandlers, Config}) ->
     lists:foreach(fun(Handler) -> Handler(Event, Labels, Config, Measurements) end, EventHandlers).
 
+-spec modules_to_funs([module()]) -> [handler_fun()].
+modules_to_funs(Modules) ->
+    [fun Module:handle_event/4 || Module <- Modules].
+
```

Comment:
Somewhat unrelated to the dynamic handlers but this little thing is a gem to me. Performance!

Reply:
I updated it after your feedback some time ago, and after checking that `gen_*` in OTP do the same.